bindings/pyroot_experimental/PyROOT/python/ROOT/__init__.py

```diff
@@ -58,3 +58,19 @@ def pythonization_impl(fn):
 import sys
 from ._facade import ROOTFacade
 sys.modules[__name__] = ROOTFacade(sys.modules[__name__])
+
+# Set import hook to be able to trigger auto-loading as appropriate
+try:
+    import __builtin__
+except ImportError:
+    import builtins as __builtin__  # name change in p3
+_orig_ihook = __builtin__.__import__
+def _importhook( name, *args, **kwds ):
```

Comment:
Can you also make the formatting match the one of the facade? E.g. no spaces after/before parenthesis here, or the same for square brackets below.